node-fs-writefile/note.txt

```diff
@@ -0,0 +1 @@
+I should buy a boat.
```

Comment:
Would you live on it? Sail around the world? Or just relax on it in your copious spare time, which you will have lots of because you'll earn so much money as a software engineer???